e2e-test/log/src/integrationTest/resources/basic-grok-e2e-pipeline-with-aws-secrets.yml

```diff
@@ -0,0 +1,20 @@
+pipeline_configurations:
+  aws:
+    secrets:
+      opensearch-sink:
+        secret_id: "opensearch-sink-basic-credentials"
```

Comment:
Did you test this using secrets generated by #3501? I see the same name is used here.

Reply:
Yes.